lib/depends.c

```diff
@@ -98,6 +98,21 @@ static rpmRC headerCheckPayloadFormat(Header h) {
     return rc;
 }
 
+static void addElement(tsMembers tsmem, rpmte te, int oc)
```

Comment:
This could use a slightly less generic name. What about `tsMembersaddElement` or `tsmsAddElement`?

Reply:
I don't care much for the name either but as it's just an internal helper, it's not worth getting stuck in a bikeshed. A tiny static internal helper doesn't deserve a long name, but I can change to addMember() if that helps.

Reply:
Tried switching that to addMember() but that ended up looking just weird. I'm going to just leave it like it is, we can always rename later if we end up needing the name for something else.